vllm/config/__init__.py

```diff
@@ -2833,11 +2833,25 @@ def __post_init__(self):
 
         if self.parallel_config.enable_dbo:
             a2a_backend = envs.VLLM_ALL2ALL_BACKEND
-            assert a2a_backend == "deepep_low_latency", \
-            "Microbatching currently only supports the deepep_low_latency "\
-            f"all2all backend. {a2a_backend} is not supported. To fix set "\
-            "the VLLM_ALL2ALL_BACKEND environment variable to "\
-            "deepep_low_latency and install the DeepEP kerenls."
+            assert a2a_backend in \
+                ["deepep_low_latency", "deepep_high_throughput"], \
+            "Microbatching currently only supports the deepep_low_latency and "\
+            f"deepep_high_throughput all2all backend. {a2a_backend} is not "\
+            "supported. To fix set the VLLM_ALL2ALL_BACKEND environment "\
+            "variable to deepep_low_latency or deepep_high_throughput and "\
+            "install the DeepEP kernels."
+
+            cudagraph_mode = self.compilation_config.cudagraph_mode
+            if cudagraph_mode.has_mode(CUDAGraphMode.PIECEWISE):
+                if cudagraph_mode.has_full_cudagraphs():
+                    cudagraph_mode = CUDAGraphMode.FULL_DECODE_ONLY
+                else:
+                    cudagraph_mode = CUDAGraphMode.NONE
+                logger.warning(
+                    "Disabling PIECEWISE cudagraphs as they are not supported "\
+                    "with dual batch overlap (dbo) currently. Setting "\
+                    "cudagraph_mode to %s.", cudagraph_mode)
+                self.compilation_config.cudagraph_mode = cudagraph_mode
```

Comment:
We have `cuda_graph_mode` set in `vllm/vllm/platforms/cuda.py` for HT as well, `if (envs.VLLM_ALL2ALL_BACKEND == "deepep_high_throughput"`. Please take a look of the order for the config